### ISSUE TITLE
Bug🐛: hub/lfedge_mgmt_hub.md links are incorrect

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/open-horizon.github.io#452]
### Describe the bug.

The hub/lfedge_mgmt_hub.md markdown page uses the full `https://` URL to internal references.  These can be changed to relative paths. For example, ` [Hub admin](../admin/multitenancy.md#user-types]`

```
./hub/lfedge_mgmt_hub.md:1. Authenticate with the Hub as a [Hub admin](https://open-horizon.github.io/docs/admin/multi_tenancy.html#user-types).
./hub/lfedge_mgmt_hub.md:4. Create the new user account (add -A if the account is for an [Organization admin](https://open-horizon.github.io/docs/admin/multi_tenancy.html#user-types)):
```

### Operating environment

all

**Comment by dipesh-rawat** [comment]
Hi @johnwalicki I happy to provide the quick fix for this issue. Could you please assign this issue to me?

**Comment by dipesh-rawat** [comment]
@johnwalicki I have implement the fix and created a draft PR #454. If you could please review the same and let me know if its good to be merged.
